agent/src/http_server/routes/get_heaviest_files.rs

```diff
@@ -0,0 +1,11 @@
+use serde::Serialize;
+
+#[derive(Serialize)]
```

Comment:
I'd suggest that you allow `serde_json` to deserialize this object
```suggestion
use serde::Serialize;

#[derive(Serialize, Deserialize)]
```

Reply:
Not here i think, i only send object with this, not receiving so i don't need to deserialize